front/src/app/styleguide/styleguide.component.html

```diff
@@ -0,0 +1,15 @@
+<div>
+    <button 
+        mat-raised-button
+        color="primary"
+    >
```

Comment:
Why 4 spaces are used here as tabulation? Change to 2 spaces - it has to be consistent across whole project (same for other html files). Change your VS Code settings